### PR TITLE
Triplesplayer screens

### PR DESCRIPTION
**Pull request by samyakshah3008**
TripleS Player screens are now completed 

- Landing page
- Video listing page
- Watch later page
- Like page
- History page
- Playlist page
- Frontend authentication screens, login and signup 

image

image

image

image

### CODE REVIEW

**Review by j24m — APPROVED** [review]
UI looks good, although you should try to make a small pr. looks good to merge .

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+import LandingPage from "./pages/landingpage/LandingPage";
+import { Routes, Route } from "react-router-dom";
+import Navbar from "./components/navbar/Navbar";
+import VideoListingPage from "./pages/video-listing-page/VideoListingPage";
+import LikePage from "./pages/likepage/LikePage";
+import HistoryPage from "./pages/historypage/HistoryPage";
+import PlaylistPage from "./pages/playlist/PlaylistPage";
+import Watchlaterpage from "./pages/watchlaterpage/Watchlaterpage";
+import LoginPage from "./pages/loginpage/LoginPage";
+import Signuppage from "./pages/signuppage/Signuppage";
```
Comment (manojasarsa) [inline src/App.js:11]: You can do a barrel export here

File: src/pages/landingpage/landingpage.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+.landing-container {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    background-color: #121825f5;
```
Comment (manojasarsa) [inline src/pages/landingpage/landingpage.css:26]: This looks pretty structured

**Review by manojasarsa — APPROVED** [review]
Good to merge!

**Pull request merged**